dags/operators/emr_spark_operator.py

```diff
@@ -41,6 +41,7 @@ class EMRSparkOperator(BaseOperator):
     airflow_bucket = environ['AIRFLOW_BUCKET']
     private_output_bucket = environ['PRIVATE_OUTPUT_BUCKET']
     public_output_bucket = environ['PUBLIC_OUTPUT_BUCKET']
+    staging_private_output_bucket = environ['STAGING_PRIVATE_OUTPUT_BUCKET']
```

Comment:
Please also add this to https://github.com/mozilla/telemetry-airflow/blob/master/Dockerfile and https://github.com/mozilla/telemetry-airflow/blob/master/Dockerfile.dev